activation/activation.go

```diff
@@ -653,12 +656,12 @@ func (b *Builder) PublishActivationTx(ctx context.Context, sig *signing.EdSigner
 
 	for {
 		b.logger.Info("broadcasting",
-			log.ZShortStringer("atx", atx.ID()),
+			zap.Object("atx", atx),
```

Comment:
Logging the whole ATX as `info` is a bit much, especially in a multi-smesher setup. How about logging only the ID as before and the whole object as `debug` log?

Reply:
It happens very sporadically (once every 2 weeks for each ID) and contains crucial information for the smesher so I'd leave it as is. Besides, it's been logged at the info in the `atx published` log - I just moved logging the contents here.

Reply:
Still, worst case scenario that one line here will print 1000+ lines of logs of mostly unusable information to the average operator. I suggest to print `atx.ID()` on info and the whole object on `debug` if someone really is interested in seeing hundreds of lines of ATX and Node IDs.

Reply:
Updated the code to only log the contents when DEBUG level is enabled.